templates/v2/analysis/render_report.py

```diff
@@ -193,6 +195,7 @@ def parse_args():
     parser.add_argument("--time-scale", type=str, default="")
     parser.add_argument("--time-event", type=str, default="")
     parser.add_argument("--num-practices", type=int, default=0)
+    parser.add_argument("--request-id", type=str, default="")
```

Comment:
`parser.add_argument("--output-dir", type=str, default="output/foo")`

Should this have a better default, like "ouptut/report"?

Reply:
We can actually remove this arg altogether and construct it from the request id